cmd/migrate/v1_0_00.go

```diff
@@ -15,18 +17,22 @@ func (v *VersionMigratorV1000) ShouldMigrate(prevVersion VersionData) bool {
 }
 
 func (v *VersionMigratorV1000) PerformMigration(rlpCfg config.RollappConfig) error {
-	fmt.Println("💈 Updating relayer configuration to match new relayer...")
-	if err := relayer.UpdateRlyConfigValue(rlpCfg, []string{"chains", rlpCfg.RollappID, "value", "extra-codecs"}, []string{"ethermint"}); err != nil {
+	fmt.Println("💈 Migrating Rollapp key...")
+	migrateRollappKeyCmd := exec.Command(consts.Executables.RollappEVM, "keys", "migrate", "--home", rlpCfg.Home+"/relayer/keys/"+rlpCfg.RollappID, "--keyring-backend", "test")
```

Comment:
doesn't `rly` have this migrate functionally? by `rly keys list` maybe?
just to avoid the dependency on the rollapp/hub binary

Reply:
it doesn't have it.